tests/test_router_bgp.rb

```diff
@@ -20,17 +20,38 @@
 require_relative 'ciscotest'
 require_relative '../lib/cisco_node_utils/bgp'
 
+XR_NOT_SUPPORTED     = 'Not supported in IOS XR'
+XR_NO_VRF_SUPPORT    = 'Feature not supported in a vrf in IOS XR'
+XR_SUPPORTED_BROKEN  = 'Supported in IOS XR - needs further work'
+
+def test_add_routerid_rdauto(asnum, vrf)
```

Comment:
note that any function starting with `test_` is assumed to be a test case by Minitest - if this is just a helper method for other test cases, please change its name.